apps/nrm/views/fr-details.html

```diff
@@ -0,0 +1,17 @@
+{{<partials-page}}
+  {{$page-content}}
+    <div class="govuk-form-group">
+        <label class="govuk-label">
+        Email
```

Comment:
translations?

Reply:
Actioned in the PR for MS-724 What are your contact details?